java/values.yaml

```diff
@@ -17,10 +17,15 @@ readinessDelay: 30
 readinessTimeout: 3
 readinessPeriod: 15
 livenessPath: '/health/liveness'
-livenessDelay: 30
+livenessDelay: 0
 livenessTimeout: 3
 livenessPeriod: 15
 livenessFailureThreshold: 3
+startupPath: '/health/liveness'
+startupDelay: 20
```

Comment:
isn't this a bit much?

Reply:
Changed to 5.  Initially thought to reduce to 0 but liveness delay was initially set to 30s in the chart which meant delay applicable to all.  Wanted to ensure that wasn't changed by introducing startup probes.  

if startupDelay set to lower or 0 and the first startup probe passes (in the 10s startupPeriod), that means liveness probe kicks in after 10s though don't think it should matter once the probe passes as applications requiring a longer period can always override. 

Startup path using same path as liveness based on what the actuator exposes.